test/helper.js

```diff
@@ -0,0 +1,21 @@
+const greetbot = require('../src/index.js');
+const user_id = process.env.SLACK_USER_ID;
+const user_dm_id = process.env.SLACK_DM_ID;
+const user_dm_name = process.env.SLACK_DM_NAME;
+const other_user_id = process.env.SLACK_OTHER_USER_ID;
+const other_dm_id = process.env.SLACK_OTHER_DM_ID;
+const other_dm_name = process.env.SLACK_OTHER_DM_NAME;
+const general_id = process.env.SLACK_GENERAL_ID;
+const other_channel_id = process.env.SLACK_OTHER_CHANNEL_ID;
+const slack_ver_token = process.env.SLACK_VERIFICATION_TOKEN;
+
+global.greetbot = greetbot;
```

Comment:
Why do you need to make them global?

Reply:
global is so that each test can pick them up, without the need to redefine/require in each file. Not sure if this was where I got it from, but similar issue: https://stackoverflow.com/questions/31282839/how-to-properly-require-modules-from-mocha-opts-file